pythonforandroid/recipes/python3/__init__.py

```diff
@@ -18,15 +19,24 @@ class Python3Recipe(GuestPythonRecipe):
         :class:`~pythonforandroid.python.GuestPythonRecipe`
     '''
 
-    version = '3.7.1'
+    version = '3.8.1'
```

Comment:
Why bumping the default version to something "cutting edge"? I think it's great that we support it, but maybe it's a bit early to bump it as a default at the same time?